### PR TITLE
LKE-8379 feat(Component): use standalone component

### PR DESCRIPTION
**Pull request by aminerhanemi**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
   "packages": {
     "": {
       "name": "@linkurious/schematics",
-      "version": "1.0.2",
+      "version": "1.0.3",
```
Comment (EricMallard) [inline package-lock.json:9]: why this change only in package-lock?

**Review by EricMallard — CHANGES_REQUESTED** [review]

File: package-lock.json (hunk 1 above)
Comment (aminerhanemi) [reply package-lock.json:9]: it was already updated in package.json , version file ....
it was updated automatically in package-lock by running npm i

**Review by justinmarsan — APPROVED** [review]
